Example/RSFormViewExampleApp/SignupExampleViewController.swift

```diff
@@ -22,18 +22,30 @@ class SignupExampleViewController: UIViewController {
     formView.viewModel = viewModel
     formView.delegate = self
     
-//    let configurator = FormConfigurator()
-//    configurator.editingLineColor = UIColor.white
-//    configurator.editingTitleColor = UIColor.white
-//    configurator.formBackgroundColor = UIColor.black
-//    configurator.fieldsBackgroundColor = UIColor.black
-//    configurator.errorTextColor = UIColor.blue
-//    configurator.invalidTitleColor = UIColor.yellow
-//    configurator.invalidLineColor = UIColor.yellow
-//    configurator.placeholderTextColor = UIColor.white
-//    configurator.validTitleColor = UIColor.lightGray
+    let configurator = FormConfigurator()
+    configurator.editingTitleColor = UIColor.astralBlue
+    configurator.editingBorderColor = UIColor.astralBlue
     
-//    formView.formConfigurator = configurator
+    configurator.formBackgroundColor = UIColor.lightGray
+    configurator.fieldsBackgroundColor = UIColor.lightGray
+    
+    configurator.errorTextColor = UIColor.yellow
+    configurator.invalidTitleColor = UIColor.yellow
+    configurator.invalidBorderColor = UIColor.yellow
+   configurator.errorTextColor = UIColor.yellow
```

Comment:
indentation crooked here. Also lets move this to a `configureFormColors` function and take care of the extra spaces here and there